src/about-compat/aboutCompat.js

```diff
@@ -6,6 +6,13 @@
 
 /* globals browser */
 
+let availablePatches;
+let showHidden = false;
+function checkHash() {
+  redraw();
+}
+window.onhashchange = checkHash;
```

Comment:
This appears to be the only use of `checkHash`, is there something stopping us from doing

```js
window.onhashchange = redraw;
```
?

Reply:
Good catch, I'll update now.